source/docs/software/sensors/digital-inputs-software.rst

```diff
@@ -146,7 +146,7 @@ Nearly all motorized mechanisms (such as arms and elevators) in FRC should be gi
             }
         }
 
-Homing an encodered mechanism
+Homing an encoded mechanism
```

Comment:
I think encodered is intentional. encoded doesn't have the same connotation.

Reply:
Yea this was one I wasn’t sure about... I did it to get opinions and because technically encodered isn’t really a word and encoded is and it seems to me like it does fit

Reply:
Even though "encodered" might not be a word per se, it can generally be understood that it means "something with an encoder on it." I don't think "encoded" has that same meaning.

Reply:
I've gotten feedback from quite a few people that just omitting the word should be fine, and I think its fine since its placed in a section about encoders anyways